src/applications/disability-benefits/526EZ/tests/helpers.unit.spec.jsx

```diff
@@ -159,6 +159,13 @@ describe('526 helpers', () => {
       const { formData: transformedPrefill } = prefillTransformer([], prefilledData);
       expect(transformedPrefill.disabilities[0].disabilityActionType).to.equal('INCREASE');
     });
+    it('should transform prefilled data when disability name has special chars', () => {
+      const newName = '//()';
+      const dataClone = _.set(_.cloneDeep(initialData), 'disabilities[0].name', newName);
+      const prefill = prefillTransformer([], dataClone, {});
+      expect(prefill.formData.disabilities[0].name).to.equal(newName);
+      expect(prefill.formData.disabilities[0].name).to.equal(newName);
```

Comment:
Is this duplicate of line 166 intentional?

Reply:
🤦‍♂️ Nope, don't think so. Nice catch!